src/app/core/views/Preferences/tabs/Plans/PlanSelector.tsx

```diff
@@ -242,13 +242,15 @@ function Price({
               amount: displayAmount(amountMonthly),
             })}
       </p>
-      {interval !== 'lifetime' && (
+      {interval !== 'lifetime' ? (
         <p className=" text-gray-50">
           {translate('views.account.tabs.plans.card.annually', {
             amount: displayAmount(amountAnnually),
           })}
         </p>
-      )}
+      ) : (
+        <p className=" text-gray-50">{translate('views.account.tabs.plans.card.oneTimePayment')}</p>
+      )}{' '}
```

Comment:
I think that Prettier has add the space `{' '}`

Reply:
Uhh, I'll remove that space.